composer.json

```diff
@@ -26,14 +26,14 @@
         "nelmio/security-bundle": "~1.4",
         "doctrine/doctrine-migrations-bundle": "~1.0.0",
         "doctrine/migrations": "~1.0.0|1.0.0@alpha",
-        "mopa/bootstrap-bundle": "3.0.0-RC2",
+        "mopa/bootstrap-bundle": "3.0.0",
```

Comment:
I didn't, it appears they have quite a few changes: https://github.com/relaxnow/MopaBootstrapBundle/pull/1

But I'm not well versed enough in the innards to decide whether this is 'safe'.

What do you think @rjkip ?